lib/muffin_man/catalog_items/v20220401.rb

```diff
@@ -0,0 +1,14 @@
+module MuffinMan
+  module CatalogItems
+    class V20220401 < V20201201
```

Comment:
If you're going to inherit here, I think it'd be better to have a base class, and then have both versions inherit from the base class.

Reply:
I have added base_api class for these API's